ludwig/models/llm.py

```diff
@@ -67,6 +71,7 @@ def __init__(
         # Used only for its metadata about the vocabulary
         self.tokenizer = AutoTokenizer.from_pretrained(self.config_obj.model_name, use_fast=False)
 
+        ####SXK what is the significance of the generation config, again just trying to understand high level context
```

Comment:
It tells the model how to generate tokens - it controls things such as the temperature, the number of new tokens, the minimum number of tokens, the top p / top k tokens to look it, whether to do greedy decoding or multinomial sampling, etc.